kpatch-build/kpatch-build

```diff
@@ -674,6 +674,10 @@ usage() {
 	echo "		                        (not recommended)" >&2
 }
 
+if ! command -v gawk &> /dev/null; then
+	die "gawk not exist, please install gawk first"
```

Comment:
Thanks.  Please change the message to "gawk not installed" for consistency with some of the other error messages.

Reply:
@jpoimboe  OK,I have modified as you said,please help review again.